### PR TITLE
WD-7790 - uptade /telco/osm page

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Update [OSM](https://ubuntu-com-13421.demos.haus/telco/osm) page according to copy doc

## QA

- [demo link](https://ubuntu-com-13421.demos.haus/)
- [copy doc](https://docs.google.com/document/d/1LQim0WRVdDZE9-ToFKSD52O7kgurMoytrgDDAPYnug8/edit#heading=h.xrvnk0etn33s)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that links and text are correct according to copy doc

## Issue / Card
[WD-7790](https://warthogs.atlassian.net/browse/WD-7790)

Fixes #

## Screenshots

[WD-7790]: https://warthogs.atlassian.net/browse/WD-7790?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/telco/osm/index.html [hunk 1]
```diff
@@ -43,8 +43,10 @@ <h1>
     <div class="col-6">
       <h2>OSM</h2>
       <h4>Open-source implementation of ETSI NFV MANO stack</h4>
-      <p>Helps Telecommunications Service Providers (TSPs) move from traditional, legacy networking services to cloud-native network functions.</p>
-      <p>Fully aligned with ETSI NFV Information Models.</p>
+      <ul>
+        <li>Helps yelecommunications service providers (TSPs) move from traditional, legacy networking services to cloud-native network functions.</li>
```
Comment (anthonydillon) [inline templates/telco/osm/index.html:47]:
```suggestion
        <li>Helps telecommunications service providers (TSPs) move from traditional, legacy networking services to cloud-native network functions.</li>
```

File: templates/telco/osm/index.html [hunk 2]
```diff
@@ -85,7 +87,7 @@ <h2>Charmed OSM</h2>
         <li class="p-list__item is-ticked">Predictable release cadence and upgrade path</li>
       </ul>
       <p>
-        <a href="https://charmed-osm.com/">Read more&nbsp;&rsaquo;</a>
+        <a href="https://charmed-osm.com/">Read more about Charmed OSM&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/telco/osm/index.html:90]: *Suggestion in the copy doc* to remove this link as this site does not exist anymore.

File: templates/telco/osm/index.html [hunk 3]
```diff
@@ -146,7 +148,7 @@ <h2>Flexible VIM options</h2>
       <p>OSM supports all the major VIMs (virtualized infrastructure manager) in the ecosystem.</p>
       <p>A single OSM instance can deploy network functions on OpenStack, AWS, VMware and k8s clusters.
       </p>
-      <p><a href="/telco/osm/multi-cloud">Multi-cloudVIM orchestration in OSM&nbsp;&rsaquo;</a>
+      <p><a href="/telco/osm/multi-cloud">Multi-cloud VIM orchestration in OSM&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/telco/osm/index.html:151]:
```suggestion
      <p><a href="/telco/osm/multi-cloud">Multi-cloud orchestration in OSM&nbsp;&rsaquo;</a>
```

File: templates/telco/osm/index.html [hunk 4]
```diff
@@ -348,10 +350,10 @@ <h3 class="p-heading--4">
 <section class="p-strip">
   <div class="row u-vertically-aligned">
     <div class="col-7">
-    <h3>Innovate in telecommunications with Canonical</h3>
-    <p>The experts in open source operations, from cloud to devices are waiting to help you on your journey.</p>
+    <h3>Innovate in telco with Canonical</h3>
+    <p>The experts in open source operations, from cloud to devices are waiting to help you.</p>
```
Comment (anthonydillon) [inline templates/telco/osm/index.html:354]:
```suggestion
    <p>The experts in open source, from cloud to devices are waiting to help you.</p>
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
@lizzochek good stuff. A few suggestions and comments in the code.

File: templates/telco/osm/index.html (hunk 2 above)
Comment (lizzochek) [reply templates/telco/osm/index.html:90]: @Sophie-32 should this link be there?

Comment (Sophie-32) [reply templates/telco/osm/index.html:90]: yes can leave off for now, PM can resubmit if they have a new link

**Comment by Sophie-32** [comment]
@lizzochek - I left two comments in the copydoc of things to update, after that its +1 from me. Thanks so much!

**Comment by lizzochek** [comment]
@Sophie-32 @anthonydillon updated

**Review by anthonydillon — APPROVED** [review]
Thanks for the updates @lizzochek

**Pull request merged**